README.md

```diff
@@ -1 +1,188 @@
-# Hello
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+  
+  <br/>
+
+  <h3><b>Hello Microverse README</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Linters Project with Github for HTML and CSS] <a name="about-project"></a>
+
+
+
+**Linters Project with Github will help you detect bugs in your html and css files**
+
+## 🛠 Built With Ojwang Briton<a name="built-with"></a>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **It's able to detect html bugs**
+- **It's able to detect css bugs**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+To install a text editor e.g "VS CODE"
+To install GIT
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Commands:
+
+```sh
+  cd hello microverse
+  git clone git@github.com:Eccb7/Hello.git
+```
+
+### Install
+
+Install this project with:
+
+Command:
+
+```sh
+  cd Hello
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Command:
+
+```
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+
+
+```
+
+```
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] As mentioned by the previous reviewer since deploying this project isn't a requirement you can remove this section or add a shot text saying "Non-deployable😊..."